cli/slsa-verifier/main.go

```diff
@@ -42,6 +66,9 @@ func main() {
 		"[optional] expected version the binary was compiled from. Uses semantic version to match the tag")
 	flag.BoolVar(&printProvenance, "print-provenance", false,
 		"print the verified provenance to std out")
+	inputs.kv = make(map[string]string)
+	flag.Var(&inputs, "workflow-input",
+		"[optional] a workflow input provided by a user at trigger time (e.g., workflow_dispatch) in the format 'key=value'.")
```

Comment:
This only supports workflow_dispatch, so e.g. should be clear that this is the only supported case

Reply:
Done, PTAL